package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kijiji-scraper",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Since the value assigned to `ad.info[attr.label]` will be changing, this could potentially break software that uses kijiji-scraper. I've been following [SemVer](https://semver.org/), which suggests that the major version number should be changed when incompatible changes are made. So the new version should be 3.0.0.